### ISSUE TITLE
DNS search suffix not for namespace on Windows pods

### ISSUE THREAD

**Issue by PatrickLang** [Azure/azure-container-networking#146]
**Is this an ISSUE or FEATURE REQUEST?** (choose one):

Issue
---

**Which release version?**:

v1.0.4-1-gf0f090e
---

**Which component (CNI/IPAM/CNM/CNS)**:

CNI
---

**Which Operating System (Linux/Windows)**:

Windows Server version 1803
---

**Which Orchestrator and version (e.g. Kubernetes, Docker)**

Kubernetes

**What happened**:

The DNS suffix is not set based on the pod's namespace.

**What you expected to happen**:

On Linux, we see the right resolution order. 
```
/etc # cat resolv.conf
nameserver 10.0.0.10
search default.svc.cluster.local svc.cluster.local cluster.local lje1130gi5iehkdbexza4ptcba.xx.internal.cloudapp.net
```

Windows only has `svc.cluster.local` 

**How to reproduce it** (as minimally and precisely as possible):

Deploy a Windows cluster with acs-engine v0.17

**Comment by saiyan86** [comment]
This is fixed in PR#152

**Comment by daschott** [comment]
@saiyan86 I see this issue again on acs-engine v0.18.9 Since you did [#152](https://github.com/Azure/azure-container-networking/pull/152) are you on point for fixing this?
